src/components/stepper/step-label/step-label.ts

```diff
@@ -0,0 +1,88 @@
+import { type CSSResultGroup, html, type TemplateResult } from 'lit';
+import { customElement } from 'lit/decorators.js';
+
+import {
+  SbbButtonBaseElement,
+  SbbDisabledTabIndexActionMixin,
+  SbbIconNameMixin,
+  hostAttributes,
+} from '../../core/common-behaviors';
+import { ConnectedAbortController } from '../../core/eventing';
+import type { SbbStepElement } from '../step/step';
+import type { SbbStepperElement } from '../stepper';
+
+import style from './step-label.scss?lit&inline';
+import '../../icon';
+
+let nextId = 0;
+
+/**
+ * Combined with a `sbb-stepper`, it displays a step's label.
+ *
+ * @slot - Use the unnamed slot to provide a label.
+ */
+@customElement('sbb-step-label')
+@hostAttributes({
+  slot: 'step-label',
+  role: 'tab',
```

Comment:
could role be part of element internals? (at least in checkbox it worked)

Reply:
I tried it and, although I see the role correctly set in the internals, the screen-reader doesn't seem to read them correctly 😕

Reply:
ok that's a bad sign. hmm which browser screenreader combination was affected?

Reply:
VoiceOver with all the browsers, I didn't test all the others.